README.md

```diff
@@ -93,6 +93,23 @@ oneview_ethernet_network 'Eth1' do
 end
 ```
 
+### oneview_fabric
+
+Fabric resource for HPE OneView.
+
+Support only in API300 onwards with variant Synergy.
+
+Action defaults to `:none`, also performs updates on the reserved vlan range.
```

Comment:
Do you envision a user ever using the `:none` action? I know it can be useful for cases when you want to notify it from another resource, but I don't know enough about fabrics to know if that's a valid use case for this resource.